src/routes/transactions/transactions.controller.spec.ts

```diff
@@ -1370,6 +1371,80 @@ describe('Transactions Controller (Unit)', () => {
         });
     });
   });
+
+  describe('Propose transactions', () => {
```

Comment:
We should split this test class into multiple files (per functionality).

The fie is close to 1.5k lines and it will get trickier to review (when we have lifecycle events that can affect the tests such as `beforeAll`, `beforeEach`, etc.)

Reply:
I totally agree. Do you think that is within the scope of this PR? I was thinking this could be done separately, I'd be happy to do it if you agree.

Reply:
> I totally agree. Do you think that is within the scope of this PR? I was thinking this could be done separately, I'd be happy to do it if you agree.

Either way is fine! 👍

Reply:
https://github.com/safe-global/safe-client-gateway-nest/issues/328